Common/src/main/java/vazkii/botania/client/render/tile/RenderTileSpreader.java

```diff
@@ -83,20 +84,36 @@ public void render(@Nonnull TileSpreader spreader, float partialTicks, PoseStack
 			ms.mulPose(Vector3f.ZP.rotationDegrees(180));
 			ms.mulPose(Vector3f.XP.rotationDegrees(180));
 			// Prevents z-fighting. Otherwise not noticeable.
-			ms.scale(0.999F, 0.999F, 1F);
+			ms.scale(0.997F, 0.997F, 1F);
 			Minecraft.getInstance().getItemRenderer().renderStatic(stack, ItemTransforms.TransformType.NONE,
 					light, overlay, ms, buffers, 0);
 			ms.popPose();
 		}
 
 		if (spreader.paddingColor != null) {
-			BakedModel model = getPaddingModel(spreader.paddingColor);
+			ms.pushPose();
+			// The padding model is facing up so that the textures are rotated the correct way
+			// It's simpler to do this than mess with rotation and UV in the json model
+			ms.translate(0.5F, 0.5F, 0.5F);
+			ms.mulPose(Vector3f.XP.rotationDegrees(-90));
+			ms.mulPose(Vector3f.YP.rotationDegrees(180));
```

Comment:
what's the purpose of this? is the padding model not rotated correctly around Y axis?

Reply:
The padding model has the "hole"/opening for the spreader at the top, instead of pointing "forwards" like the spreader model itself. This is so the wool texture faces in a consistent direction on all the faces. As mentioned in the comment, it's possible to do this by rotating and plotting in UV values in the model itself, but that's annoying and error-prone. when the "hole" is pointing up, the textures on all the 4 sides around it are facing the same direction (up), and look consistent. (If you make the hole on one of the sides, like it is currently/before this pr, textures will be pointing in different directions). Images: (Red lines are to illustrate the "flow"/direction of the wool texture)

Old:
![image](https://user-images.githubusercontent.com/39034386/154865827-e7d41acd-ed2c-428f-8b96-14492cc2f79e.png)

New:
![image](https://user-images.githubusercontent.com/39034386/154865862-e09cbf50-0ed6-477a-8806-d1e44a2442d9.png)

Reply:
oh i was more confused about the Y 180 rotation specifically
the whole sub-branch of code i understand why it's there 
but to go pointing up -> pointing forward just needs an x rotation right?

Reply:
it needs the y rotation because: lining up all the textures requires rotating either the "front" face or the "back" face (really top and bottom in the model) 180 degrees. By just doing the x rotation, it turns out the face that needs the rotation to be upright when the spreader is facing forwards, is the front/`up` face. but that face is on 4 different cuboids and requires tweaking the UV mapping to rotate around, whlie the back face can be made to be on just 1 cuboid. so it's easier to rotate that and then rotate the whole model 180deg